Classes/foxPlayerInput.uc

```diff
@@ -305,6 +315,7 @@ defaultproperties
 	Desired43FOV=90f
 	bCorrectZoomFOV=true
 	bCorrectMouseSensitivity=true
+	bCorrectWeaponFOV=True

```

Comment:
Similar to above, I would move this up 1 line to sit under "bCorrectZoomFOV", so that the FOV options are grouped together.